File: Data Structures & Algorithms/remove-element/submission-3.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        curr_count = 0
        eq_val = 0
        for i, num in enumerate(nums):
            if i >= len(nums)-eq_val:
                break
            if num == val:
                swap = True
                while swap and i != len(nums)-eq_val:
                    eq_val += 1
                    nums[i] = nums[len(nums)-eq_val]
                    if nums[i] != val:
                        swap = False
        return len(nums)-eq_val

File: Data Structures & Algorithms/max-consecutive-ones/submission-3.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        int max_count = 0;
        int curr_count = 0;
        for (int i = 0; i < nums.Length; ++i) {
            curr_count = (nums[i] == 0) ? 0 : curr_count + 1;
            max_count = Math.Max(max_count, curr_count);
        }
        return max_count;
    }
}